README.md

```diff
@@ -1,50 +1,55 @@
 # sssp-protocol
-This repository contains all the necessary workflows and tools in order to run the SSSP (standard solid state pseudopotential) protocol for testing pseudopotentials.
+This repository contains all the necessary workflows and tools in order to run
```

Comment:
For git reasons, it's nicer to have complete single sentences on a single line (as it was).